src/main/groovy/com/devsoap/vaadinflow/extensions/VaadinFlowPluginExtension.groovy

```diff
@@ -73,6 +74,7 @@ class VaadinFlowPluginExtension {
         productionMode = project.objects.property(Boolean)
         compatibilityMode = project.objects.property(Boolean)
         submitStatistics = project.objects.property(Boolean)
+        whitelistedPackages = project.objects.property(String[])
```

Comment:
To be forward compatible with Gradle 6 we will need to use ``project.objects.listProperty(String)`` instead